src/PythonClient/src/quixstreams/dataframes/platforms/quix/config.py

```diff
@@ -265,7 +264,7 @@ def get_confluent_broker_config(self, known_topic: Optional[str] = None) -> dict
 
     def get_confluent_client_configs(
         self, topics: list, consumer_group_id: Optional[str] = None
-    ):
+    ) -> tuple:
```

Comment:
Could you pls specify what types this tuple contains? 
You can do it with `typing.Tuple` class